### ISSUE TITLE
Rent Adjustment of Low Residential Housing

### ISSUE THREAD

**Issue by jangoergens** [Jimmyokok/LandValueOverhaul#2]
My low residential homes have rents of 50$/month which seems pretty low to me. Row housing or medium density housing is usually 3-4 times more expensive, which still seems way too cheap. I think low residential housing in a good neighborhood should be at least 500$/month.

**Comment by Jimmyokok** [comment]
I understand your concern. However, my response in issue #1 also fits here. The game is currentlly not well designed and very far from being a reality simulation game. You can't play this game with common sense.

**Comment by jangoergens** [comment]
Is there a way to see how much individuals make? I remember the rent being 3000$/month before installing the mod, which caused all but a select few citizens to move out of their homes. If the income is indeed too low to even support 500$/month, then I see no issues with the current rent and you can close this issue.

**Comment by Jimmyokok** [comment]
There is no such way. However, citizen wages depend on their education levels. The actual values of these wages are fixed and are not observable even via code search. Some players have calculated them with developer mode, and the following are their results:

- Uneducated: 1200
- Poorly educated: 2000
- Educated: 2500
- Well educated: 3500
- Highly educated: 5000

Meanwhile, there is another way to estimate household income. Rent update is computed according to household willing-to-pay rent, which < 45% household income. You mentioned a 3000 rent when playing the original game, which means 50% households have willing-to-pay rent > 3000, and you may safely estimate that 50% households are making > (3000 /0.45 = 6667) every month.

While I agree with you that the rent should be much higher in reality, there are two points here, excluding the wage issue:

- First, the game's logic is _renter income determines rent _, instead of _rent determines renter_, which is more close to the reality. The consequence is the huge variance of income among renter households. In your case, where 50% households are making > 6667 every month, a significant proportion of households could be making much less than that every month.
- And, this mod is a balance between the game's original mechanism and the game experience of players. The original game is literally unplayable, as the rent will eventually stabilize at a level where 50% renters want to move away. This mod makes the rent update mechanism favors the poor, satisfying _most_ renters instead of 50%. Given the variance of willing-to-pay rent, the eventual rent should be significantly lower than in the original game, and consequently, there are fewer citizens moving away and almost no high-rent warnings.

In short, this mod ensures game experience while sacrificing reality simulation, and the game's wage setting is why these two cannot be simultaneously preserved.

**Comment by jangoergens** [comment]
Okay I see your point, but I think the rent price should not be set to the lowest denominator. I would argue, that in highly desirable areas, rent prices should be higher and uneducated and poorly educated citizens should be moving out (I am trying to get them to leave my city altogether, since there are not jobs for them in high level industrial buildings).

Is it possible to add an option in the menu to change this behavior? I obviously don't want the vanilla implementation, but never having to rezone/rebuild certain areas also doesn't quite feel right.

**Comment by Jimmyokok** [comment]
I think your demand makes sense, but things can be extremely complicated this way. Since the minimum (reasonable) unit for citizen management is actually household, and households can be composed of citizens of different education levels, adding an option to drive away all citizens of a certain education level can be quite unrealistic.
A better way to achieve this is to make another mod, which somehow unbounds rent and land value and implements  _rent determines renter_. This would prevent low income citizens (=low education level citizens in the game) from moving into high rent areas and I believe this could be the ultimate solution to your problem.

**Comment by jangoergens** [comment]
True, that would be the best solution (hopefully in vanilla some day), for the meantime I am simply destroying buildings with uneducated citizens in them and hope for others to move in. Thanks for creating this mod!
